Harvest.Core/Services/SlothService.cs

```diff
@@ -127,8 +142,7 @@ public async Task<SlothResponseModel> MoveMoney(int invoiceId)
                 var credit = await _financialService.IsValid(expense.Key);
```

Comment:
If an account isn't valid and it is preventing processing, maybe we should throw off an email too.

Reply:
yea agree with this, otherwise there'll be no way to find it.  not sure the best way though because we don't want to send out emails every day

Reply:
It is possible that the validation will fail because of KFS issues.
Maybe trust them here, but have a job that validates accounts once a week or so? @srkirkland

Reply:
don't really love an update job idea but maybe instead we just trust them always since Brian's team is doing billing review anyway?  And he can note when accounts need to be changed?  Or alternately we can do an invoice status of accountInvalid and then put up some big warning if there are any in that state. Maybe create a ticket automatically if any accounts are invalid

Reply:
Yeah, probably just let sloth/Brian deal with it.